addon/styles/global-rules.less

```diff
@@ -0,0 +1,194 @@
+.ember-appointment-slots-pickers {
+  li {
+    list-style: none;
+  }
+  .white {
+    color: @white;
+  }
+  .background-dark-blue {
+    background-color: @dark-blue;
+  }
+  .border-white {
+    border-color: @white;
+  }
+  .border-right {
+    border-right-style: solid;
+    border-right-width: 1px;
+  }
+  .border-light-grey {
+    border-color: @light-grey;
+  }
+  .font-size-1 {
+      font-size: 50px;
+      line-height: 56px
+  }
+
+  @media (min-width: 480px) {
+    .font-size-1 {
+        font-size:70px;
+        line-height: 76px
+    }
+  }
+
+  .font-size-2 {
+      font-size: 40px;
+      line-height: 46px
+  }
+
+  @media (min-width: 480px) {
+      .font-size-2 {
+          font-size:50px;
+          line-height: 56px
+      }
+  }
+
+  .font-size-3 {
+      font-size: 30px;
+      line-height: 36px
+  }
+
+  @media (min-width: 480px) {
+      .font-size-3 {
+          font-size:40px;
+          line-height: 46px
+      }
+  }
+
+  .font-size-4 {
+      font-size: 30px;
+      line-height: 36px
+  }
+
+  .font-size-5 {
+      font-size: 22px;
+      line-height: 28px
+  }
+
+  .font-size-6 {
+      font-size: 20px;
+      line-height: 26px
+  }
+
+  .font-size-7 {
+      font-size: 18px;
+      line-height: 24px
+  }
+
+  .font-size-8 {
+      font-size: 16px;
+      line-height: 22px
+  }
+
+  .font-size-9 {
+      font-size: 14px;
+      line-height: 20px
+  }
+
+  .font-size-10 {
+      font-size: 13px;
+      line-height: 19px
+  }
+
+  .font-size-11 {
+      font-size: 12px;
+      line-height: 18px
+  }
+
+  .font-size-phone {
+      font-size: 26px;
+      line-height: 32px
+  }
+
+  .bold {
+      font-weight: 700
+  }
+
+  .regular {
+      font-weight: 400
+  }
+
+  .light {
+      font-weight: 300
+  }
+
+  .p0 {
+    padding: 0
+  }
+
+  .p1 {
+      padding: 5px
+  }
+
+  .py1 {
+    padding-top: 5px;
+    padding-bottom: 5px;
+  }
+
+  .p2 {
+      padding: 10px
+  }
+
+  .p3 {
+      padding: 15px
+  }
+
+  .p4 {
+      padding: 20px
+  }
+
+  .p5 {
+      padding: 25px
+  }
+
+  .p6 {
+      padding: 30px
+  }
+
+  .p7 {
+      padding: 35px
+  }
+
+  .p8 {
+      padding: 40px
+  }
+
+  .p9 {
+      padding: 45px
+  }
+
+  .mb2 {
+    margin-bottom: 10px;
+  }
+
+  .border-radius-top4 {
+    border-radius: 4px 4px 0 0;
+  }
+
+  .btn {
+    font-size: 20px;
+    font-weight: 700;
+    //-webkit-border-radius: 40px;
+    //-moz-border-radius: 40px;
+    border-radius: 40px;
+    padding: 7px 30px;
+    border: 0;
+    box-shadow: none;
+    display: inline-block;
+  }
+
+  .btn-secondary {
+    color: #005eb8;
+    background: 0 0;
+    border: 2px solid #005eb8;
+    padding: 5px 28px;
+    text-decoration: none;
+  }
+
+  .btn-primary {
+    color: #333f48;
+    background: #b3e000;
```

Comment:
Are there variables for these colours?

Reply:
@NikkiAlgarCentrica this is a PR for an open-source addon, so can't use bg-vi - so will need to tree-shake out this CSS when importing into  connectedHomes repo (or put the CSS in the dummy app).

I guess those should be added to https://github.com/britishgas-engineering/ember-appointment-slots-pickers/blob/feature/transfer-slot-pickers-to-open-source/addon/styles/variables.less though, I will add it as a bullet point if that's fine.

Reply:
@NikkiAlgarCentrica fixed thanks